R/pipe_consistency_linter.R

```diff
@@ -0,0 +1,79 @@
+#' Pipe consistency linter
+#'
+#' Check that pipe operators are consistently used throughout a file. Optionally
+#' specify which pipe operator to use.
+#'
+#' @param pipe Which pipe operator to use (either `"%>%"` or `"|>"`). By default
+#' (`"auto"``), the linter has no preference but will check that the same pipe
+#' operator is used throughout the file.
```

Comment:
throughout each file?

Not sure how to best express that auto only looks at one file at a time, i.e. won't find inconsistencies that are split across files.

Reply:
> will check that each file only uses one type of pipe

maybe?